plugins/theme-manager/ThemeManager.css

```diff
@@ -0,0 +1,35 @@
+:root {
+  --border-style: 1px solid rgba(0, 0, 0, 0.1);
+}
+
+.container {
+  display: flex;
+  height: 100%;
+  width: 100%;
+  position: absolute;
+}
+
+.list {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  min-width: 220px;
+  max-width: 33%;
+  border-right: var(--border-style);
+  overflow-y: auto;
+}
+
+.listItem {
+  padding: 5px;
+  border-bottom: var(--border-style);
+}
+
+.listItem:last-child {
+  border-bottom: 0;
+}
+
+.document {
+  margin: 0;
+  padding: 0 10px;
+  max-width: 66%;
+}
```

Comment:
I don't think we need this CSS file. Please use [Sanity UI](https://www.sanity.io/ui)